cmd/backup/encrypt_archive.go

```diff
@@ -9,6 +9,7 @@ import (
 	"os"
 	"path"
 
+	"filippo.io/age"
```

Comment:
Question: is it also possible to use the import path `https://github.com/FiloSottile/age` for this package? I do understand the one you use here is the canonical one, but I am slightly worried about specifying a domain here that is under the responsibilty of a single person. Credit cards expire, people go to jail, my build breaks, that kind of stuff. If it's not possible, it's not a blocker.